lib/resources/r4/appointment.yaml

```diff
@@ -186,3 +186,25 @@ fields:
       ]
     }
   url: https://hl7.org/fhir/appointment-definitions.html#Appointment.requestedPeriod
+
+- name: reasonCode
+  required: 'No'
+  type: List of CodeableConcept
+  description: Coded reason this appointment is scheduled.
+  action: create
+  note: Must contain a text field.
```

Comment:
`reasonCode.text` is required.

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/063202ecf7b52c89b017b9b8d20412808d417975#diff-fbb215a73c336ca1ac34d91de2fb411eR76)

Reply:
backtick it

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/74138237b033684b95e9ebd89bbd163b23aa2d59#diff-fbb215a73c336ca1ac34d91de2fb411eL76)

Reply:
updated again [here](https://github.com/cerner/fhir.cerner.com/commit/923e21eef058f2b4164f3ffc6340bebae8db0aca#diff-fbb215a73c336ca1ac34d91de2fb411eL76)